packages/paste-core/layout/flex/src/index.tsx

```diff
@@ -60,24 +81,47 @@ const Flex = React.forwardRef<HTMLDivElement, FlexProps>(
       () => getFlexStyles({basis, hAlignContent, grow, shrink, vertical, vAlignContent, wrap}),
       [basis, hAlignContent, grow, shrink, vertical, vAlignContent, wrap]
     );
+
+    if (size && (width || height)) {
+      console.error('[Paste Flex]: you cannot set height or width with the size attribute.');
```

Comment:
I really liked the wording of "you cannot set a  ________ when using the ______ attribute(s)" that was used in the other error messages, and I think it makes sense to use it here as well. 
```suggestion
      console.error('[Paste Flex]: you cannot set a height or width when using the size attribute.');
```

Reply:
Agree, but I'll push it up separately